ansible_collections/arista/avd/examples/l2ls-fabric/README.md

```diff
@@ -0,0 +1,137 @@
+# Example for L2LS Fabric
+
+## Introduction
+
+This example is meant to explore using AVD with an L2LS (Layer 2 Leaf Spine) Topology. The example topology shown below contains 2 Spines and 2 Leaf Pairs. The L2LS Fabric is purely Layer 2 with no routing of SVIs in the Fabric.  Later we will add in L3 routing functionality to this example`.
+
+![Figure: Arista L2LS physical topology](images/pure_L2LS_topo.png)
+
+## AVD Setup
+
+- Define Ansible Inventory (Hierarchical Groupings)
+- Define AVD Fabric input variables (group_vars/host_vars)
+
+### Sample Ansible Inventory
+
+```yaml
+DC1:
+  children:
+    DC1_FABRIC:
+      children:
+        DC1_SPINES:
+          hosts:
+            SPINE1:
+              ansible_host: 172.100.100.101
+            SPINE2:
+              ansible_host: 172.100.100.102
+        DC1_LEAFS:
+          children:
+            POD1:
+              hosts:
+                LEAF1:
+                  ansible_host: 172.100.100.105
+                LEAF2:
+                  ansible_host: 172.100.100.106
+            POD2:
+              hosts:
+                LEAF3:
+                  ansible_host: 172.100.100.107
+                LEAF4:
+                  ansible_host: 172.100.100.108
+    DC1_NETWORK_SERVICES:
+      children:
+        DC1_LEAFS:
+        DC1_SPINES:
+    DC1_NETWORK_PORTS:
+      children:
+        DC1_LEAFS:
+        DC1_SPINES:
+```
+### Sample AVD group_vars/host_vars
+
+In the ansible variables, you need to set the following required variables:
+
+- fabric_name = name of the fabric, must match a group name in your ansible inventory (ie. DC1_FABRIC)
+- design.type = l2ls
+- spine node type keys
+- leaf node type keys
```

Comment:
```suggestion
In the Ansible variables,  the following variables are required:

- fabric_name = name of the fabric, must match a group name in your Ansible inventory (i.e. DC1_FABRIC)
- design.type = l2ls
- spine node type keys
- leaf node type keys
```

Reply:
Hey @gmuloc. I am still in draft mode of writing the docs here.  Appreciate the comments.  I was rewriting much of this  today.   Please take a look now and see how it reads.  I plan to mark this PR ready for review next week.